index.js

```diff
@@ -1,30 +1,79 @@
 const express = require("express");
 const app = express();
+const fs = require('fs');
 
-// Finally got this line to work, so /static leads to any necessary static content. ~ SixBeeps
-app.use('/static', express.static(__dirname + '/static'));
-app.use('favicon.ico', express.static(__dirname + '/favicon.ico'))
+app.use("/static", express.static(__dirname + "/static"));
 
-app.get("/", async(req, res) => {
-  console.log("Yeeet, [DEBUG] we running in home page");
-  res.sendFile(`${__dirname }/views/home.html`);
+app.use('/js', express.static(__dirname + '/js'));
+
+app.use("favicon.ico", express.static(__dirname + "/favicon.ico"));
```

Comment:
We discussed about removing this

Reply:
I don't know...You can do it. i don't know exactly which line you are referring to.